### PR TITLE
speedy: add R4, merge db-blankdraft to G13

### LINKED ISSUE #466: Add new deletion-tagging template {{Db-r4}}
A discussion <https://en.wikipedia.org/wiki/Wikipedia_talk:Criteria_for_speedy_deletion#Redirects_in_the_File:_namespace_G6_→_R4> found consensus to use new deletion tag {{db-r4}}. It should only be accesible in File: namespace.

### PR DESCRIPTION
**Pull request by siddharthvp**
Added newly-introduced CSD R4 per [discussion](https://en.wikipedia.org/wiki/Special:PermanentLink/872246394#Redirects_in_the_File:_namespace_G6_%E2%86%92_R4). Merged db-blankdraft to db-g13 per [discussion](https://en.wikipedia.org/w/index.php?title=Wikipedia_talk:Criteria_for_speedy_deletion&oldid=872669362#Proposal_to_merge_db-blankdraft_into_G13)

Closes #466.

### CODE REVIEW

File: modules/twinklespeedy.js [hunk 1]
```diff
@@ -1055,6 +1055,11 @@ Twinkle.speedy.redirectList = [
 		value: 'redirtypo',
 		tooltip: 'However, redirects from common misspellings or misnomers are generally useful, as are redirects in other languages'
 	},
+	{
```
Comment (Amorymeltzer) [inline modules/twinklespeedy.js]: This should really be in the `fileList` no?

File: modules/twinklespeedy.js [hunk 2]
```diff
@@ -790,17 +790,17 @@ Twinkle.speedy.userNonRedirectList = [
...
-		hideWhenMultiple: true
-	},
 	{
 		label: 'G11: Promotional user page under a promotional user name',
 		value: 'spamuser',
 		tooltip: 'A promotional user page, with a username that promotes or implies affiliation with the thing being promoted. Note that simply having a page on a company or product in one\'s userspace does not qualify it for deletion. If a user page is spammy but the username is not, then consider tagging with regular G11 instead.',
 		hideWhenMultiple: true
+	},
+	{
+		label: 'G13: Old, abandoned AfC submission or blank draft',
```
Comment (QEDK) [inline modules/twinklespeedy.js]: Drafts unedited in 6 months are also included, not only blank ones, modify the label as well. Change value to g13 to preserve uniformity with the other rationales imo.

File: modules/twinklespeedy.js [hunk 3]
```diff
@@ -790,17 +790,17 @@ Twinkle.speedy.userNonRedirectList = [
...
+		value: 'afc',
+		tooltip: 'Any rejected or unsubmitted AfC submission, or a blank draft, unedited in over 6 months,',
```
Comment (QEDK) [inline modules/twinklespeedy.js]: This implies only blank drafts are eligible which is not true.

**Comment by Amorymeltzer** [comment]
Tests seem good!  I've been in template/module hell trying to confirm where db-multiple seems to support R4...

**Comment by Amorymeltzer** [comment]
@JJMC89 Was just editing to clarify that I meant what my import busted on testwiki, while it works just fine on enwiki.

File: modules/twinklespeedy.js (hunk 2 above)
Comment (siddharthvp) [reply modules/twinklespeedy.js]: Doesn't the label already say "Old, abandoned AfC submission" which means a draft unedited in 6 months? Anyways, please post your suggested wording here - I'll make the change.

File: modules/twinklespeedy.js (hunk 3 above)
Comment (siddharthvp) [reply modules/twinklespeedy.js]: Like above, I don't see how the wording implies that. What wording are you suggesting?

File: modules/twinklespeedy.js (hunk 2 above)
Comment (QEDK) [reply modules/twinklespeedy.js]: @siddharthvp Simply make it Old, abandoned AfC submissions and draftspace pages and for the tooltip write the same including pages with no content, unedited in more than 6 months.

Comment (Amorymeltzer) [reply modules/twinklespeedy.js]: So theoretically this particular message will only show up on user pages, so it's okay not to clarify the draftspace content.  It's probably worth updating the other one though.

**Comment by siddharthvp** [comment]
@Amorymeltzer @QEDK I've completely changed the labels and tooltips of G13 in both userList and generalList, to avoid any confusion. Note that in userList, it's not necessary to mention draftspace pages. And in generalList, it's not worth mentioning blank drafts in the label because Article Wizard-generated blank drafts occur only in userspace, and the userList criteria are anyway shown above generalList criteria  if the page is indeed in userspace.

**Comment by Amorymeltzer** [comment]
@siddharthvp Thanks for the speedy updates.  This request got lost amid the other comments, but can you deal with the `'blankdraft': 'g6',` line in `normalizedhash`?

**Pull request merged**